data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+			`	He is silent for a moment, with the air of someone who is enjoying not having to run for the first time in a while. "Oh, Tali said that you might have some historical records for us?" You hand over the copy of the archive in response, telling him that is probably the best available. He hands you a credit chip with <payment> on it. "Thank you, I'll make sure this gets where it needs to go. In the meantime, we could use your help in the starport."`
+
+
+
+mission "Remnant: Salvage 3"
+	source "Viminal"
+	name "Transport injured Remnant"
+	description "Some injured Remnant need to be transported away from the front lines to Aventine."
+	passengers 8
+	cargo "Medical gear" 5
+	blocked "A prefect informs you that they need to evacuate some injured to a hospital on Aventine, but you will need 8 free bunks and 5 tons of cargo space in order to handle the injured."
+	stopover Aventine
+	to offer
+		has "Remnant: Salvage 2: done"
+	on offer
+		conversation
+			`After watching the mechanics start repairs on <ship>, you make the short trek to the spaceport common area. People are still rushing around as ships continue to land. No wrecks so far, and judging by the scrolling arrivals displays there are still quite a few ships on approach. Looking around for someone to talk to, you spot the prefect who had just visited your ship. He notices you and comes over. "No ships lost yet," he signs harshly, "but there are a few ships that haven't checked in yet."`
+			`	He shakes his head and continues, "We need to keep our medical facilities as free as possible. Could you take some of our injured to Aventine? They are stable and healthy enough for an unattended transport, and it would free up medics and space here."`
+			choice
+				`	"Of course."`
+				`	"I'd rather not."`
+					decline
+			`	He looks relieved. "Thanks. I was going to have to divert one of our sentries to take them." He pauses and consults his comlink. "The medical team will have the injured settled onboard shortly, along with everything they need to ensure they arrive safely. They will probably have some fresh crew to send back with you."`
+				accept
+	on stopover
+		event "Remnant Albatross"
+		dialog
+			`The injured crew make for an eerily quiet jaunt from Viminal to Aventine. They don't sing, and rarely sign. Some spend the trip mellowed out on powerful painkillers, while the few that leave their rooms seem to be mostly intent on doing prescribed stretches and movements to help damaged muscles heal. A couple are interested in visiting with someone from outside the Ember Waste, but they lack the energy to do so for long.`
+			`	On Aventine there are ambulances and medical staff waiting to transport them to longer-term medical facilities. As they depart, a fresh crew arrives to take their places.`
```

Comment:
Since this is a new paragraph, and especially considering that we have introduced new subjects into the dialogue, it should be specified who is being transported here.

Reply:
I'm not sure exactly what you mean about breaking the paragraph.

I did clarify the "them" though.